From: SJ Park <sj@kernel.org>
Date: Mon, 31 Aug 2026 07:07:53 -0700
Subject: [PATCH] ==== misc cleanups ====

mm/damon: misc cleanups

Cleanup the code, tests and samples for clarifications and readability.

The patches are individually sent by the authors.  I'm reposting those
as one series for convenience of handling.  For this reason, changelog
is on each patch's commentary section.

Changes from v1
- v1: https://lore.kernel.org/20260831142611.77572-1-sj@kernel.org
- None.

Signed-off-by: SJ Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_4ehcsi_g | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_4ehcsi_g

diff --git a/hkml_cv_bogus/hkml_cv_bogus_4ehcsi_g b/hkml_cv_bogus/hkml_cv_bogus_4ehcsi_g
new file mode 100644
index 0000000000000..e69de29bb2d1d
-- 
2.47.3

